### DISCUSSION TITLE
"Fixed" vs. "Random" Effect for TauU_Meta

### DISCUSSION THREAD

**Discussion by gjones1219** [jazznbass/scan#85] [Q&A] [answered]
Hello! I hope you are taking care. I was wondering if I can get some more clarity on what "fixed" vs. "random" effects for TauU_Meta means and how it impacts the analyses. Based on my understanding of "fixed" vs. "random" effects, random effects are estimated by partial pooling of all of the data, meaning that the most abundant data will have the greatest impact on the results for TauU_Meta in this case. The study I am using employs a multiple, non-concurrent baseline design, meaning that some participants will arbitrarily have more data than others; this fact means that a "random" effect would unfairly overweight their results for TauU_Meta. Therefore, I am inclined to use a "fixed" effect for TauU_Meta within my study, but wanted to check in with you before I did so to get more background on "fixed" vs. "random" for single case experiments. I am also inclined to go with "fixed" because the power analyses in Scan (upon which I based my study design) are also designed around fixed effects as well. Any guidance here would be appreciated, thank you!

Grant

**Comment by gjones1219** [comment]
Following up here, thank you!

**Reply by gjones1219** [reply]
Following up again, thanks.

**Comment by jazznbass** [comment, marked as answer]
Hi Grant,
Pooling in a meta-analysis is based on the standard error of the single cases and not on the abundance of the data. The result you get is not the average effect of all cases, but the total effect of all data in terms of the effects studied.
This is a crucial difference. Meta-analysis does not represent the results that an average individual would show. Rather, it is concerned with the effects found in the totality of all cases as a unit of description.
The difference between "fixed" and "random" effects has implications for the interpretation of your data.
With fixed effects, you assume that the cases analyzed are all the cases to which the results should apply. That is, the statistics are based on just those cases and the conclusions you draw are valid for those cases.
For random effects, you assume that your cases are a random sample from a pool of all cases to which your results should apply. That is, the statistics are an estimate for the effects in the population of your sample.
I hope, this helps!
Jürgen
